docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,37 @@
+# Grid Layout
+
+## Overview
```

Comment:
Kindly remove this heading to maintain consistency across files.

Reply:
Done.